module/surveySubmissions/helper.js

```diff
@@ -737,4 +730,70 @@ module.exports = class SurveySubmissionsHelper {
         })
     }
 
+
+   /**
+   * Get survey submission details
+   * @method
+   * @name surveySubmissionDetails
+   * @param {String} submissionId - survey submissionId
+   *  @param {String} status - survey submission status.
+   * @returns {JSON} - survey submission details
+   */
+
+    static surveySubmissionDetails(submissionId, status = "") {
+        return new Promise(async (resolve, reject) => {
+            try {
+                let queryObject = {
+                    _id: submissionId
+                }
+
+                if ( status != "" ) {
+                    queryObject.status = status;
+                }
+
+                let surveySubmissionsDocument = await this.surveySubmissionDocuments( queryObject );
+    
+                if (!surveySubmissionsDocument.length) {
+                    throw messageConstants.apiResponses.SUBMISSION_NOT_FOUND;
+                }
+
+                let solutionDocument = await solutionsHelper.solutionDocuments({
+                    _id: surveySubmissionsDocument[0].solutionId
+                }, [ "name","scoringSystem","description","questionSequenceByEcm"]);
+    
+                if(!solutionDocument.length){
+                    throw messageConstants.apiResponses.SOLUTION_NOT_FOUND;
+                }
+                
+                solutionDocument = solutionDocument[0];
+                surveySubmissionsDocument[0]['solutionInfo'] = solutionDocument;
+
+                if ( surveySubmissionsDocument[0] && surveySubmissionsDocument[0].programId && surveySubmissionsDocument[0].programId != "" ) {
```

Comment:
@VISHNUDAS-tunerlabs remove this first check
`surveySubmissionsDocument[0]`
because its already checked on top